design-documents/graph-ql/coverage/reward-points.graphqls

```diff
@@ -0,0 +1,38 @@
+type Customer {
+    reward_points: RewardPoints @doc(description: "Customer reward points")
+}
+
+type RewardPoints {
+    balance: RewardPointsBalance @doc(description: "Current reward points balance")
+    exchange_rate: RewardPointsExchangeRate @doc(description: "Current reward points exchange rate")
+    subscription_status: RewardPointsSubscriptionStatus @doc(description: "Reward points related email subscription status")
+    balance_history: [RewardPointsBalanceHistoryItem] @doc(description: "Reward points balance history. Visibility of this section depends on the configuration")
```

Comment:
> Visibility of this section depends on the configuration

This comment doesn't make it clear whether the type definition will not exist in the schema based on config, or if it'll just be `null` depending on config

Reply:
Thank you, please check adjusted description.

Reply:
Thanks! New description is much more clear